contracts/extensions/MultisigPermissions.sol

```diff
@@ -400,14 +400,14 @@ contract MultisigPermissions is
           } else {
             emit RejectionChanged(msgSender(), _motionId, roleIndex, _setTo);
           }
-        }
 
-        if (_vote == Vote.Approve) {
-          uint256 threshold = getDomainSkillRoleThreshold(motion.domainSkillId, roleIndex);
-          if (motionVoteCount[_motionId][_vote][roleIndex] < threshold) {
-            anyBelowThreshold = true;
-          } else if (motionVoteCount[_motionId][_vote][roleIndex] == threshold && _setTo) {
-            newlyAtThreshold = true;
+          if (_vote == Vote.Approve) {
```

Comment:
This could be moved up into the previous block.

Reply:
Done, which I believe also means we can keep the name of `newlyAtThreshold`?